CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## v1.0.2
+### 2024-2-26
+
+This version of HOSS correctly handles edge-aligned geographic collections by 
+adding the attribute `cell_alignment` with the value `edge` to `hoss_config.json`
+for edge-aligned collections (namely, ATL16), and by adding functions that 
+create pseudo bounds for edge-aligned collections to make HOSS use the 
+`dimension_utilities.py` function, `get_dimension_indices_from_bounds`.
```

Comment:
Given that these are our public release notes, it might be worth explicitly stating in some way that the pseudo-bounds variables are not returned in the actual HOSS output.